content/chapters/software-stack/lab/solution/basic-syscall/arm/Makefile

```diff
@@ -10,6 +10,9 @@ hello: hello.o
 
 hello.o: hello.s
 
+run:
+	qemu-aarch64 -L /usr/aarch64-linux-gnu ./hello
```

Comment:
I would add a `QEMU` variable initialized to `qemu-aarch64`.

Reply:
Why is `-L` required? It works without `-L` on my side.

Reply:
I had left the `-L` just in case (for robustness), removed it, now, as per your suggestion.